docs/platform/tutorials/image-processing/get-project.md

```diff
@@ -0,0 +1,86 @@
+# Get the project
+
+While you can try out the live demo, or experiment using the ungated product experience, it can be useful to learn how to get a project up and running in Quix. 
+
+Once you have the project running in your Quix account, you can modify the project as required, and save your changes to your forked copy of the project. With a forked copy of the repository, you can also receive upstream bug fixes and improvements if you want to, by syncing the fork with the upstream repository. 
+
+In the following sections you learn how to:
+
+1. Fork an existing project repository, in this case the image processing template project.
+2. Create a new project (and environment) in Quix linked to your forked repository.
+
+In later parts of the tutorial you explore the project pipeline using the Quix data explorer and other tools, viewing code, examining data structures, and getting a practical feel for the Quix Portal.
+
+## 💡 Key ideas
+
+The key ideas on this page:
+
+* Forking a public template project repository
+* Connecting Quix to an external Git repository, in this case the forked repository
+* Quix projects, environments, and applications
+* Pipeline view of project
+* Synchronizing an environment
+
+## Fork the project repository
+
+Quix provides the image processing template project as a [public GitHub repository](https://github.com/quixio/computer-vision-demo){target="_blank"}. If you want to use this template as a starting point for your own project, then the best way to accomplish this is to fork the project. Using fork allows you to create a complete copy of the project, but also benefit from future bug fixes and improvements by using the upstream changes.
+
+To fork the repository:
+
+1. Navigate to the [Quix GitHub repository](https://github.com/quixio/computer-vision-demo){target="_blank"}.
+
+2. Click the `Fork` button to fork the repo into your GitHub account (or equivalent Git provider if you don't have a GitHub account). Make sure you fork all branches, as you will be looking at the `develop` branch.
+
+    !!! tip 
+
+        If you don't have GitHub account you can use another Git provider, such as Bitbucket. On Bitbucket you could import the repository.
+
+## Create your Quix project
+
+Now that you have a forked copy of the repository in your GitHub account, you can now link your Quix account to it. Doing this enables you to build and deploy the project in your Quix account, and examine the pipeline much more closely.
+
+To link Quix to this forked repository:
+
+1. Log into your Quix account.
+
+2. Click `+ Create project`.
+
+3. Give your project a name. For example, "Computer Vision".
```

Comment:
These all could be condensed into one step with instructions separated by commas.